tests/test_utils.py

```diff
@@ -15,62 +15,59 @@
 def test_compute_nfw_boost() :
     """Test the nfw model for boost factor"""
     # Test data
-    rvals = np.arange(1,11)
+    rvals = np.arange(1,11).tolist()
```

Comment:
are these `.tolist()` added here necessary?

Reply:
It is not necessary. I added it for consistency because all other tests use lists as the input, so we can check if lists are supported. The old tests failed to capture the fact that the relevant functions could not handle lists as the input.

Reply:
ok, good idea